### PR TITLE
Add new datasets in Dec. 2021

### PR DESCRIPTION
**Pull request by PaulWessel**
This updates the _earth_relief_ data, adds the related _earth_synbath_, and then add new data sets for Earth (faa, geoid, vgg, mag, mag4km, and wdmam). Note: I have not tried to build the docs.

### CODE REVIEW

**Comment by meghanrjones** [comment]
Docs look good.

We should hold off on merging until they are placed on the server. Just to confirm - the terms allow all these to be re-distributed?

**Comment by PaulWessel** [comment]
Yes, I checked with both mag groups and got their preferred CPTs and David is onboard of course.
The Gebco grid is placed in the public domain so we do not need to request any approval but we may want to get their preferred CPT if any.  I started downloading their grid last night but timed out...

**Comment by PaulWessel** [comment]
Great @Esteban82, can you make a PR for dcw-gmt? That way we can help with fixing these and other spelling issues form the shape file.

**Comment by PaulWessel** [comment]
Hi @meghanrjones might you be able to build the updated branch so we can have a look at what might be missing if anything.  I think I have added all the data set descriptions and images.

**Comment by meghanrjones** [comment]
> Hi @meghanrjones might you be able to build the updated branch so we can have a look at what might be missing if anything. I think I have added all the data set descriptions and images.

Yes, I'll try to get Vercel set up for previews.

File: docs/index.rst [hunk 1]
```diff
@@ -6,5 +6,10 @@
 
     earth-relief.rst
     earth-age.rst
+    earth-geoid.rst
```
Comment (meghanrjones) [inline docs/index.rst:9]:
```suggestion
    earth-gebco.rst
    earth-geoid.rst
```

**Comment by PaulWessel** [comment]
@meghanrjones, might you be able to edit earth-relief.rst and replace the XXX with a valid link to the auto-resolution place in the GMT docs? I can then use that in other places and finish up those RST files.

File: docs/earth-gebco.rst [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+06m      3600 x   1800 g,p  8.0 MB  6 arc minute global relief (GEBCO 2021 @ 10 km)
+05m      4320 x   2160 g,p   11 MB  5 arc minute global relief (GEBCO 2021 @ 9 km)
+04m      5400 x   2700 g,p   17 MB  4 arc minute global relief (GEBCO 2021 @ 7.5 km)
+03m      7200 x   3600 g,p   30 MB  3 arc minute global relief (GEBCO 2021 @ 5.6 km)
+02m     10800 x   5400 g,p   65 MB  2 arc minute global relief (GEBCO 2021 @ 3.7 km)
+01m     21600 x  10800 g,p  237 MB  1 arc minute global relief (GEBCO 2021 @ 1.9 km)
+30s     43200 x  21600 g,p  864 MB  30 arc second global relief (GEBCO 2021 @ 1.0 km)
+15s     86400 x  43200 p    2.9 GB  15 arc second global relief (GEBCO 2021)
+03s    432000 x 216000 g    6.8 GB  3 arc second global relief (SRTM3S)
+01s   1296000 x 432000 g     41 GB  1 arc second global relief (SRTM1S)
```
[the remark this answers is not in the document]
Comment (PaulWessel) [reply docs/earth-gebco.rst:48]: Yes, same SRTM tiles. When we use the new auto-resolution syntax @earth_gebco then the resolution depends on -R -J etc and I dont think we want to be in a situation where we go from 15s to 3s and the script will complain there is no such resolution.  Note: There are no exatra files. GMT will replace earth_gebco with eart_relief for 3s and1s.

File: docs/earth-faa.rst [hunk 3]
```diff
@@ -0,0 +1,64 @@
+IGPP Global Earth Free-Air Anomaly
+----------------------------------
+.. figure:: /_static/igpp.png
+   :align: right
+   :scale: 20 %
+
+.. figure:: /_static/GMT_faafig.jpg
```
[the remark this answers is not in the document]
Comment (PaulWessel) [reply docs/earth-faa.rst]: Yes, that is a bit silly.  I just changed all to not have fig in them.

**Review by meghanrjones — APPROVED** [review]

**Pull request merged**